src/frontend/packages/cloud-foundry/src/features/cf/user-invites/user-invite.service.ts

```diff
@@ -143,13 +143,15 @@ export class UserInviteService {
       map(v => v.entity.metadata && v.entity.metadata.userInviteAllowed === 'true')
     );
 
-    this.canConfigure$ = combineLatest(
+    this.canConfigure$ = combineLatest([
       waitForCFPermissions(this.store, this.activeRouteCfOrgSpace.cfGuid),
-      this.store.select('auth')
-    ).pipe(
-      map(([cf, auth]) =>
+      this.store.select('auth'),
+      cfEndpointService.endpoint$
+    ]).pipe(
+      map(([cf, auth, endpoint]) =>
         cf.global.isAdmin &&
-        auth.sessionData['plugin-config'] && auth.sessionData['plugin-config'].userInvitationsEnabled === 'true')
+        auth.sessionData['plugin-config'] && auth.sessionData['plugin-config'].userInvitationsEnabled === 'true' &&
```

Comment:
"When registering an endpoint we show the option to 'share' it with other users. user specific endpoints should not be shared. this should be hidden if user is not an admin"

I changed it because of the first two sentences in the quote above (from your first reply to me in the issue). I assumed that the third sentence was still aimed at stratos roles and not cf roles. I will change it back.